sound/soc/sof/control.c

```diff
@@ -353,6 +353,66 @@ int snd_sof_bytes_ext_put(struct snd_kcontrol *kcontrol,
 	return 0;
 }
 
+
+int snd_sof_bytes_ext_volatile_get(struct snd_kcontrol *kcontrol,
+			  unsigned int __user *binary_data,
```

Comment:
same here. use 100 chars and keep to 2 lines

Reply:
Done